### DISCUSSION TITLE
Can I Join Team?

### DISCUSSION THREAD

**Discussion by ThakurRahulSingh** [ROM-builders/temporary#15551] [Q&A]
My github Profile: https://github.com/ThakurRahulSingh

My Telegram: https://t.me/RahulSinghBhadoriya

My Successful Booted Builds:
https://cirrus-ci.com/build/6239284361953280
https://cirrus-ci.com/build/5949905437458432

My Device Tree: Branch pex-13
DT Common: https://github.com/ThakurRahulSingh/device_xiaomi_sm6250-common
DT: https://github.com/ThakurRahulSingh/device_xiaomi_miatoll
VT Common: https://github.com/ThakurRahulSingh/vendor_xiaomi_sm6250-common
VT: https://github.com/ThakurRahulSingh/vendor_xiaomi_miatoll

Thanks

**Comment by Apon77** [comment]
please check telegram message
